### ISSUE TITLE
Notepad++ hangs while opening remote files

### ISSUE THREAD

**Issue by shabbir4791** [notepad-plus-plus/notepad-plus-plus#1831]
While opening large remote files(size>10MB), Notepad++ hangs, There is no visual feedback to the user that the file is loading. Also, user is unable to switch to other open files while the remote file is loading.

[20160506_122345.zip](https://github.com/notepad-plus-plus/notepad-plus-plus/files/251646/20160506_122345.zip)

**Comment by dail8859** [comment]
You mean remote files as in network drive or using the FTP plugin?

**Comment by shabbir4791** [comment]
Happening for files in a network drive

**Comment by dail8859** [comment]
Then I guess #913 would cover this then.

**Comment by shabbir4791** [comment]
Yes. That's the exact issue I'm facing

**Comment by mirh** [comment]
Nope, that's only part of the issue. 
A proper fix for this should also address #1866.

**Comment by MetaChuh** [comment]
please update your notepad++ to the latest version [<kbd>Notepad++ 7.6.6</kbd>](https://notepad-plus-plus.org/download/v7.6.6.html) and migrate or reinstall your plugins if you update from version 7.5.9 or below.
if your issue still persists, and it is still of relevance to you, please feel free to file a new issue as easy to understand and reproducible as possible.

thank you for your support and best regards.

-----
```
general notes:

please close your issues as soon as you are satisfied, to help
keeping the notepad++ issue tracker clean.

when creating a new issue, please provide all information as shown at the issue template.
your notepad++ debug info is part of the minimum requirements.
(? > Debug Info... > Copy debug info into clipboard).

please also conduct a search at the issue tracker beforehand, to check if a similar issue
already exists.

for general questions, or if it is not sure, whether your issue is directly related to
the notepad++ source code, please visit us at the notepad++ community forum, and
search if related topics exists. you are welcome to post either at similar
topics, or to create a new topic at Help Wanted or General Discussion.
(no extra account is needed, just use your github account to sign in)
```
[<p align="center"><kbd> &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; Click here to visit the Notepad++ Community Forum &emsp; &emsp; &emsp; &emsp; &emsp; &emsp;&emsp; &emsp; &emsp; &emsp; &emsp; &emsp; &emsp; </kbd></p>](https://notepad-plus-plus.org/community/)
